From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 17 May 2011 17:37:31 +0100
Subject: ocaml: Really fix 'make install' rule.

This fixes commit b8ad15031cacf910634b4f4f4632232949c4acd2
and commit f408b757b1d75429fae5fa7630a4fc5451844de7.
---
 ocaml/Makefile.am |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/ocaml/Makefile.am b/ocaml/Makefile.am
index 58fa8e3..c688df0 100644
--- a/ocaml/Makefile.am
+++ b/ocaml/Makefile.am
@@ -88,19 +88,19 @@ include .depend
 SUFFIXES = .cmo .cmi .cmx .ml .mli .mll .mly
 
 # Do the installation by hand, because we want to run ocamlfind.
+install_files = META *.so *.a *.cma *.cmi *.mli
+
+if HAVE_OCAMLOPT
+install_files += *.cmx *.cmxa
+endif
+
 install-data-hook:
 	mkdir -p $(DESTDIR)$(OCAMLLIB)
 	mkdir -p $(DESTDIR)$(OCAMLLIB)/stublibs
 	$(OCAMLFIND) install \
 	  -ldconf ignore -destdir $(DESTDIR)$(OCAMLLIB) \
-	  hivex \
-	  META *.so *.a *.cma *.cmi *.mli
-if HAVE_OCAMLOPT
-	$(OCAMLFIND) install \
-	  -ldconf ignore -destdir $(DESTDIR)$(OCAMLLIB) \
-	  hivex \
-	  *.cmx *.cmxa
-endif
+	  $(PACKAGE_NAME) \
+	  $(install_files)
 
 CLEANFILES += $(noinst_DATA)
 
-- 
